src/main/java/bio/terra/cli/command/resource/create/AwsS3StorageFolder.java

```diff
@@ -0,0 +1,59 @@
+package bio.terra.cli.command.resource.create;
+
+import bio.terra.cli.command.shared.WsmBaseCommand;
+import bio.terra.cli.command.shared.options.ControlledResourceCreation;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.serialization.userfacing.input.CreateAwsS3StorageFolderParams;
+import bio.terra.cli.serialization.userfacing.input.CreateResourceParams;
+import bio.terra.cli.serialization.userfacing.resource.UFAwsS3StorageFolder;
+import bio.terra.workspace.model.StewardshipType;
+import picocli.CommandLine;
+
+/**
+ * This class corresponds to the fourth-level "terra resource create aws-s3-storage-folder" command.
+ */
+@CommandLine.Command(
+    name = "aws-s3-storage-folder",
```

Comment:
Let's go with `s3-storage-folder`, as the other resource types don't prepend `gcp-`

Reply:
Done, removed for the resource name in Commands. and resource Type in CLI. 
Retained the "Aws-" prefix in internal code/classes

@jmczerk - this is the case in WSM. Would you prefer this be changed in WSM as well?